std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -490,9 +493,10 @@ struct KRRegion(ParentAllocator = NullAllocator)
             assert(pnode && pnode.next);
             assert(pnode != n);
             assert(pnode.next != n);
+
             if (pnode < pnode.next)
             {
-                if (pnode >= n || n >= pnode.next) continue;
+                if (pnode > n || n > pnode.next) continue;
```

Comment:
what the heck was that???

Reply:
That if was confusing, because memory that we want to deallocate should not be in the list (check the asserts above).